### DISCUSSION TITLE
CLI showing internal .json

### DISCUSSION THREAD

**Discussion by vkoopmans-resoluut** [ingrammicro/puzzle-tokens#55] [Q&A]
Since the last 2 Updates, I am seeing (what I think is) the internal .json that is created for Sketch in my Command Line Interface when running Puzzle Tokens through SketchTool. I don't know if that is bad or not :)

<img width="712" alt="Screenshot 2022-05-24 at 08 43 46" src="https://user-images.githubusercontent.com/40267611/169970060-ef2220a5-2da4-47eb-bf56-bb4e36273496.png">

**Comment by MaxBazarov** [comment]
Some kind of debug log. I'm trying to find a source :)
